packages/smooth_app/lib/pages/product/add_new_product_page.dart

```diff
@@ -469,4 +479,67 @@ class _AddNewProductPageState extends State<AddNewProductPage>
         disabled: disabled,
         isLoggedInMandatory: widget.isLoggedInMandatory,
       );
+
+  Widget _buildDoneCard(final BuildContext context,
+      {required final VoidCallback onDoneClick}) {
+    final AppLocalizations localizations = AppLocalizations.of(context);
+    final bool isDark = Theme.of(context).brightness == Brightness.dark;
+
+    return SizedBox(
+      width: double.infinity,
+      child: SmoothCard(
+          clipBehavior: Clip.antiAliasWithSaveLayer,
+          color: Theme.of(context).colorScheme.secondary,
+          child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: <Widget>[
+                Padding(
+                    padding: const EdgeInsets.all(SMALL_SPACE),
+                    child: Text(
+                      localizations.new_product_done_msg(
+                          OpenFoodAPIConfiguration.globalUser?.userId ??
+                              localizations.user_profile_title_guest),
```

Comment:
"user_profile_title_guest" will just be `Welcome!`

The sentence 

Thanks for your contribution "welcome!"! doesn't make sense.
I'd suggest to create a new no user string in  app_en.arb

Something like

`"new_product_done_msg_no_user": "Thanks for your contribution !",`